change/@fluentui-style-utilities-60215204-2c5b-469a-9aef-fc22d60bacc0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32241&commitId=29f34fe0cdeaf5cfc8d11452f3c7aae8daf8a43f&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
**charting visual regressions:** resolved as noise, the PR does not change anything in the charting components, one of the reported regressions does not look different, the other has some pixels in the chart curve in the diff but does not look to be caused by the files touched in this PR

**persona visual regression:** the baseline actually had a broken image, and the new rev shows a profile pic. resolved that one as correct.